scriptmodules/emulators/mupen64plus.sh

```diff
@@ -19,7 +19,7 @@ rp_module_flags="sdl2"
 
 function depends_mupen64plus() {
     local depends=(cmake libsamplerate0-dev libspeexdsp-dev libsdl2-dev libpng-dev libfreetype6-dev fonts-freefont-ttf libboost-filesystem-dev)
-    isPlatform "rpi" && depends+=(libraspberrypi-bin libraspberrypi-dev)
+    isPlatform "videocore" && depends+=(libraspberrypi0 libraspberrypi-dev)
```

Comment:
I believe libraspberrypi-dev has a dependency on libraspberrypi0 so this isn't needed.

Reply:
The `libraspberrypi-bin` dependency was added in 79c5d096eea37437c7acba1a697a6b1f816c6471 -

>    The `mupen64plus/mupen64plus.sh` script uses the `tvservice` binary and
>    in some systems this package might not be installed by default.

Reply:
Ok, I removed the spurious dependency and left `libraspberrypi-dev` only.
As for the `mupen64plus.sh` start-up script, it does indeed uses `tvservice` to determine the audio port when using the OMX audio plugin, but it should be installed automatically when `runcommand` dependencies are added.